exercises/crypto-square/description.md

```diff
@@ -22,13 +22,13 @@ is normalized to:
 "ifmanwasmeanttostayonthegroundgodwouldhavegivenusroots"
 ```
 
-The plaintext should be organized in to a rectangle. The size of the
-rectangle should be decided by the length of the message.
+The plaintext should be organized into a rectangle as square as possible.
```

Comment:
I don't think we need "as square as possible". The criteria for the rectangle follow immediately below.

Reply:
I found those criteria confusing without a clear problem statement. It lead me to considering the low-level implementation of meeting these constraints rather than focusing on the nature of the problem more abstractly.

If the idea is that people are to unravel the problem statement as well as solve the problem, then I concur, but it appears to come at the cost of being unhelpful, particularly toward beginners.

Reply:
I hear what you're saying.

I suppose saying `c = ceil(sqrt(length))` is a bit too big of a hint.

Reply:
I like having the requirement explained intuitively before giving the formal criteria 👍

Reply:
Thank you @jiegillet for distilling my thoughts. :)

I was striving for an idiomatic raku solution without explicit iterator variables. At least two of the three given constraints can be met [without direct congruence checking](https://exercism.org/tracks/raku/exercises/crypto-square/solutions/habere-et-dispertire).

So I find that to be part of the origin of my concern. The focus on the `r` and `c` criteria felt somewhat like a distracting procedural detail when the problem statement was under-defined for me searching for an orthogonal solution.

Reply:
@glennj are you okay with this phrasing, given the above discussion?

Reply:
I think the "as square as possible" _could_ help and is unlikely to cause any harm.

Reply:
That's where I've landed on this one as well.